src/Language/Python/Internal/Render.hs

```diff
@@ -209,13 +209,101 @@ intToHexH n = go n []
     go 15 = (++[HeXDigitF])
     go b = let (q, r) = quotRem b 16 in go r . go q
 
-renderPyCharsBytes :: QuoteType -> StringType -> [PyChar] -> Text
-renderPyCharsBytes qt st =
-  case st of
-    LongString ->
-      Text.pack . go . correctBackslashes . correctInitialFinalQuotes qt
-    ShortString ->
-      Text.pack . go . correctBackslashes . correctQuotes qt
+renderPyCharsWithCorrection
+  :: (QuoteType -> StringType -> [PyChar] -> [PyChar])
+  -> QuoteType
+  -> StringType
+  -> [PyChar] -> Text
+renderPyCharsWithCorrection c qt st = Text.pack . go . c qt st
+  where
+    go s =
+      case s of
+        [] -> ""
+        Char_newline : cs -> "\\newline" <> go cs
```

Comment:
This this meant to be `\\newline` or `\\n`?

Reply:
It is correct - `\newline` is an escape sequence in Python. See the table at the end of this paragraph https://docs.python.org/3.5/reference/lexical_analysis.html#string-and-bytes-literals